streams/src/test/java/org/apache/kafka/streams/state/internals/CompositeReadOnlyKeyValueStoreTest.java

```diff
@@ -64,8 +64,9 @@ public void before() {
 
         theStore = new CompositeReadOnlyKeyValueStore<>(
             new WrappingStoreProvider(Arrays.<StateStoreProvider>asList(stubProviderOne, stubProviderTwo)),
-                                        QueryableStoreTypes.<String, String>keyValueStore(),
-                                        storeName);
+            QueryableStoreTypes.<String, String>keyValueStore(),
```

Comment:
is the kafka way to align this with the `(` .

Reply:
It's not fixed in the style guide. The only thing is says is either to put all params on a single line or put each param on a separate line.

What I've seen (which I think makes sense) is, if the line's not too long, and it doesn't hinder readability, you can either do this:

```java
myMethod(one
         two,
         three);
```

or

```java
myMethod(
  one,
  two,
  three
);
```

Experience says that either one might be more readable in different circumstances, so it makes sense to allow both. They're both equally obvious regarding where all the params are and where the method call ends. The latter one uses more vertical space for simple method calls, but is more friendly when some of the arguments are themselves method calls.

Reply:
Thanks for the explanation. I was more curious about this, than anything.. Personally, the latter is very natural and easy to do. I just go with the flow, in these things. :)

Reply:
Agreed. I also prefer the latter. Thanks for asking! It's way easier to review code that already conforms to the norm, whether or not it's required by the style guide.